src/test/resources/wiremock/mappings/mapping-historicalPriceEmptySymbol.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name" : "historical_price_empty_symbol",
+  "request" : {
+    "url" : "/stock/chart/max/{date}?symbol=&range=max",
+    "method" : "GET"
+  },
+  "response" : {
+    "status" : 200,
+    "jsonBody" : [{
+      "symbol": "Invalid"
```

Comment:
Is this how the IEX api will actually respond when you pass symbol as blank? 200 with symbol: "Invalid"? Remember, wiremock should be matching the behavior of the IEX API when possible, so that the test is capturing actually possible scenarios when the app is running against the real IEX API.

Reply:
Yes, I was trying to generate an error message but will have to look into how the IEX API actually works